addons/stock/static/src/js/counted_quantity_widget.js

```diff
@@ -1,36 +1,42 @@
-/** @odoo-module alias=stock.counted_quantity_widget **/
-
-import BasicFields from 'web.basic_fields';
-import fieldRegistry from 'web.field_registry';
-
-const CountedQuantityWidgetField = BasicFields.FieldFloat.extend({
-    supportedFieldTypes: ['float'],
-
-     _renderReadonly: function () {
-        if (this.recordData.inventory_quantity_set) {
-            this.el.textContent = this._formatValue(this.recordData.inventory_quantity);
-        } else {
-            this.el.textContent = "";
-        }
-    },
+/** @odoo-module **/
+
+import { FloatField } from "@web/views/fields/float/float_field";
+import { registry } from "@web/core/registry";
+
+const { useEffect, useRef } = owl;
+
+export class CountedQuantityWidgetField extends FloatField {
+    setup() {
+        // Need to adapt useInputField to overide onInput and onChange
+        super.setup()
+
+        const inputRef = useRef("numpadDecimal" || "input");
```

Comment:
It's strange, it's the same than doing `useRef("numpadDecimal")` as the condition is `String1 or String2`, it will always take the first string